src/kademlia/dht_tracker.cpp

```diff
@@ -316,6 +316,20 @@ namespace libtorrent { namespace dht {
 			n.second.dht.announce(ih, listen_port, flags, f);
 	}
 
+	void dht_tracker::sample_infohashes(udp::endpoint const& ep, sha1_hash const& target
+		, std::function<void(time_duration
+			, int, std::vector<sha1_hash>
+			, std::vector<std::pair<sha1_hash, udp::endpoint>>)> f)
```

Comment:
I feel like this function taking another function may warrant a comment to how it is used